### PR TITLE
Docker wordpress test

### PR DESCRIPTION
**Pull request by MurruAlessio**
The following changes were made to adapt the italiaWP2 theme and the Onelogin plugin to our test case. In particular:
- **ItaliaWP2 theme**: creation of the welcome page with the logged-in user's information
- **Onelogin plugin**: modified the settings and configuration file to apply the configurations not reachable from the graphic menu and to allow the saving of the spidCode field obtained from IdP

### CODE REVIEW

File: example/onelogin_custom_settings/settings.php [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        'logoutResponseSigned' => $opt['logoutResponseSigned'] == 'on'? true: false,
+        'wantMessagesSigned' => $opt['wantMessagesSigned'] == 'on'? true: false,
+        'wantAssertionsSigned' => $opt['wantAssertionsSigned'] == 'on'? true: false,
+        'wantAssertionsEncrypted' => $opt['wantAssertionsEncrypted'] == 'on'? true: false,
+        'wantNameId' => false,
+        'requestedAuthnContext' => $opt['requestedAuthnContext'],
+        'relaxDestinationValidation' => true,
+        'lowercaseUrlencoding' => get_option('
+            onelogin_saml_advanced_idp_lowercase_url_encoding', false),
+        'signatureAlgorithm' => get_option('onelogin_saml_advanced_signaturealgorithm', 'http://www.w3.org/2000/09/xmldsig#rsa-sha1'),
```
Comment (peppelinux) [inline example/onelogin_custom_settings/settings.php:135]: use sha256 instead of sha1

File: example/onelogin_custom_settings/settings.php [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        'digestAlgorithm' => get_option('onelogin_saml_advanced_digestalgorithm', 'http://www.w3.org/2000/09/xmldsig#sha1'),
```
Comment (peppelinux) [inline example/onelogin_custom_settings/settings.php:136]: the same previous comment

File: example/satosa/disco.html [hunk 3]
```diff
@@ -148,7 +148,7 @@ <h3>IT Wallet</h3>
                     <div class="row">
                       <div class="col-lg-12 text-center button-container column-buttons">
                           <!-- IDEM IDP BUTTON SMALL "ENTRA CON WALLET ID" * begin * -->
-                          <a href="#" class="btn btn-primary btn-lg btn-me w-100" eidas-idp-button="#eidas-idp-button-xlarge-post" aria-haspopup="false" aria-expanded="false"  style="font-size: 21px;">
+                          <a href="/qr-code-page.html" class="btn btn-primary btn-lg btn-me w-100" eidas-idp-button="#eidas-idp-button-xlarge-post" aria-haspopup="false" aria-expanded="false"  style="font-size: 21px;">
```
Comment (peppelinux) [inline example/satosa/disco.html:151]: Non riesco a capire perché le modifiche non vengano testate prima del commit

questo era https://github.com/italia/Satosa-Saml2Spid/blob/eudi/example/static/disco.html#L186

e fino a prova contraria deve rimanere in quella maniera altrimenti la chiamata di ritorno al proxy non funziona.
Su questo problema siete rimasti bloccati per giorni e vi ho sbloccato io con i miei commit, perché adesso stiamo regredendo?

@Pyscho79 ^

**Review by peppelinux — CHANGES_REQUESTED** [review]

File: example/README.md [hunk 4]
```diff
@@ -43,4 +43,27 @@ After following these steps, your WordPress instance should be up and running wi
...
+- **Sign AuthnRequest**: `true`
+- **Reject Unsigned Assertions**: `true`
+- **NameIDFormat**: `urn:oasis:names:tc:SAML:2.0:attrname-format:uri`
+- **requestedAuthnContext**: `urn:oasis:names:tc:SAML:2.0:ac:classes:X509`
+- **Service Provider X.509 Certificate**: insert the X.509 certificate of your SP
+- **Service Provider Private Key**: insert the private key of your SP
+- **Signature Algorithm**: rsa-sha256
+- **Digest Algorithm**: sha256
+
+Once all fields are set, save the settings and download the SP metadata for its configuration on IAM Proxy.
```
Comment (ruphy) [inline example/README.md:69]:
```suggestion
To configure your test environment with the IAM Proxy instance, you'll need to undertake a configuration phase on the OneLogin plugin settings page. The required proxy service configuration metadata is obtainable from https://demo-it-wallet.westeurope.cloudapp.azure.com/Saml2IDP/metadata. 

Specifically, you should modify the following fields:

- **IdP Entity Id**: Enter the entityID of the IAMProxy, which can be located within the metadata.
- **Single Sign-On Service Url**: Input the Location of the SingleSignOnService you desire to connect with, as specified in the metadata file.
- **X.509 Certificate**: Include the X.509 Certificate associated with the IAMProxy, found within the metadata.
- **Create user if not exists**: Set this to `true`.
- **Update user data**: Set this to `true`.
- **Attribute Mapping - Username**: Set this to `fiscalNumber`.
- **Attribute Mapping - E-mail**: Set this to `urn:oid:1.2.840.113549.1.9.1.1`.
- **Attribute Mapping - First Name**: Set this to `Name`.
- **Attribute Mapping - Last Name**: Set this to `familyName`.
- **Service Provider Entity Id**: Enter the URL of your SP metadata as the entityID. For example: http://\<wordpress-domain\>/wp-login.php?saml_metadata
- **Encrypt nameID**: Set this to `true`.
- **Sign AuthnRequest**: Set this to `true`.
- **Reject Unsigned Assertions**: Set this to `true`.
- **NameIDFormat**: Set this to `urn:oasis:names:tc:SAML:2.0:attrname-format:uri`.
- **requestedAuthnContext**: Set this to `urn:oasis:names:tc:SAML:2.0:ac:classes:X509`.
- **Service Provider X.509 Certificate**: Insert your SP's X.509 certificate here.
- **Service Provider Private Key**: Input the private key of your SP.
- **Signature Algorithm**: Set this to `rsa-sha256`.
- **Digest Algorithm**: Set this to `sha256`.

After you've filled all the fields, save your settings and download the SP metadata for configuration on the IAM Proxy.
```

File: example/onelogin_custom_settings/settings.php (hunk 1 above)
Comment (MurruAlessio) [reply example/onelogin_custom_settings/settings.php]: resolved with [9047963c1abd51a8505bc09abb07f86f0f387cc3](https://github.com/italia/eudi-wallet-it-python/pull/75/commits/9047963c1abd51a8505bc09abb07f86f0f387cc3)

File: example/onelogin_custom_settings/settings.php (hunk 2 above)
Comment (MurruAlessio) [reply example/onelogin_custom_settings/settings.php]: resolved with [9047963c1abd51a8505bc09abb07f86f0f387cc3](https://github.com/italia/eudi-wallet-it-python/pull/75/commits/9047963c1abd51a8505bc09abb07f86f0f387cc3)

File: example/satosa/templates/qrcode.svg
[the remark this answers is not in the document]
Comment (MurruAlessio) [reply example/satosa/templates/qrcode.svg:1]: resolved with https://github.com/italia/eudi-wallet-it-python/pull/75/commits/2b65b84a24461a51927d5f6e0ac9655a787c326c

File: example/satosa/disco.html (hunk 3 above)
Comment (MurruAlessio) [reply example/satosa/disco.html]: resolved with https://github.com/italia/eudi-wallet-it-python/pull/75/commits/920db32c73c124860a51ff216134970e34519ae4

**Review by peppelinux — APPROVED** [review]

**Pull request merged**